apps/aecore/lib/aecore/chain/worker.ex

```diff
@@ -191,7 +192,7 @@ defmodule Aecore.Chain.Worker do
       Persistence.write_block_by_hash(block)
 
       ## Block was validated, now we can send it to other peers
-      Peers.broadcast_to_all({:new_block, block})
+      Client.broadcast_block(block)
```

Comment:
I'd like to divide the client as much from the rest of the code as possible. Only the Peers Worker should know about the HTTP-Client Interface